### PR TITLE
Remove the `hive` prefix from general filesystem config properties

### PR DESCRIPTION
**Pull request by alexjo2144**
WIP: Needs associated docs change

## Description

Since these Config classes are reused across Hive, Iceberg, and Delta Lake, remove the `hive` prefix from their properties.

> Is this change a fix, improvement, new feature, refactoring, or other?

Renaming properties only, no functional changes

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)

Connectors which use these filesystem properties

> How would you describe this change to a non-technical end user or system administrator?

Hopefully should make the naming of these properties less confusing when using them from non-Hive connectors.

## Documentation

Not done yet, but is needed.

( ) No documentation is needed.
( ) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

( ) No release notes entries required.
(x) Release notes entries required with the following suggested text:

...

### CODE REVIEW

**Comment by alexjo2144** [comment]
Making this a draft until I get a chance to update the docs

File: plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/DeltaLakeQueryRunner.java [hunk 1]
```diff
@@ -110,10 +110,10 @@ public static DistributedQueryRunner createS3DeltaLakeQueryRunner(
                 coordinatorProperties,
                 extraProperties,
                 ImmutableMap.<String, String>builder()
-                        .put("hive.s3.aws-access-key", MINIO_ACCESS_KEY)
-                        .put("hive.s3.aws-secret-key", MINIO_SECRET_KEY)
-                        .put("hive.s3.endpoint", minioAddress)
-                        .put("hive.s3.path-style-access", "true")
+                        .put("aws.s3.aws-access-key", MINIO_ACCESS_KEY)
+                        .put("aws.s3.aws-secret-key", MINIO_SECRET_KEY)
```
Comment (findepi) [inline plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/DeltaLakeQueryRunner.java:114]: I would be tempted to call this just `s3.xxxx`.
did you consider that?

Comment (alexjo2144) [reply plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/DeltaLakeQueryRunner.java:114]: I did, but I liked having the convention across all three being `cloud.service.property_name`.

Comment (findepi) [reply plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/DeltaLakeQueryRunner.java:114]: Makes sense to me.

@electrum would you mind taking a look here?

**Comment by mosabua** [comment]
This is a good usability improvement .. we just need to make sure we have it all as legacy config so things wont break and we update all the docs.  And then as next step we can clean up metastore;-)

**Pull request closed without merging**